redskyctl/internal/commands/experiments/get.go

```diff
@@ -276,10 +276,12 @@ func (o *GetOptions) filterAndSortTrials(l *experimentsv1alpha1.TrialList) error
 
 // sortableTrialData slightly modifies the schema of the trial item to make it easier to specify sort orders
 func sortableTrialData(item *experimentsv1alpha1.TrialItem) map[string]interface{} {
-	assignments := make(map[string]int64, len(item.Assignments))
+	assignments := make(map[string]interface{}, len(item.Assignments))
```

Comment:
Should we use numstr here instead?

Reply:
We need to pass JSON primitives down into the sorting code.